__tests__/app.test.js

```diff
@@ -124,4 +133,26 @@ describe("Articles endpoint", () => {
         expect(body.text).toBe("Article Not Found");
       });
   });
+  test("POST:201 - Posts a comment to the specified article", () => {
+    const newPost = { username: "butter_bridge", body: "TL;DR!" };
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(newPost)
+      .expect(201)
+      .then((result) => {
+        expect(result.body.comment.body).toEqual(newPost.body);
+        expect(result.body.comment.author).toEqual(newPost.username);
+        expect(result.body.comment.article_id).toBe(2);
+      });
+  });
+  test("POST:400 - Responds with an error message when given insufficient post data", () => {
+    const newPost = { username: "butter_bridge" };
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(newPost)
+      .expect(400)
+      .then((result) => {
+        expect(result.body.msg).toBe("Bad Request");
+      });
+  });
```

Comment:
A few tests missing here.  It's a prism of endless possibility, where a single choice can branch out into infinite realities, creating alternate worlds from the ones you know. I am the Watcher. I am your guide through these vast new realities. Follow me and ponder the question... "What if?"

- What if the article doesn't exist? - `/api/articles/1000/comments`
- What if the article id is not valid? - `/api/articles/banana/comments`
- What if the value in newPost is of the wrong datatype? - `{username: true, body: "did this work?"}`
- What if the username doesn't exist? - `{username: "Lando-Norris", body: "It's Friday theeen!"}`